live-examples/js-examples/string/string-raw.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js">const name = 'Bob';
+
+// The next line returns the template with the
+// value of the name variable inserted.
```

Comment:
This isn't exactly right - the next line logs a string to the console.

If possible, please use the "expected output:" format to label console output, as per https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#indicating-consolelog-output:

```
console.log(String.raw`<p>Good morning, ${name}</p>`);
// expected output: "<p>Good morning, Bob</p>"
```